packages/core/src/hooks/useActiveDescendantListFocus/__stories__/useActiveDescendantListFocus.stories.tsx

```diff
@@ -14,6 +14,7 @@ type Story = StoryObj<typeof useActiveDescendantListFocus>;
 
 export default {
   title: "Hooks/useActiveDescendantListFocus",
+  // @ts-ignore
```

Comment:
how come those kind of stuff that you've added ignore in many places didn't bug us until today?

Reply:
Because the stories file migrated to ts
This is a bug we need to resolve but not for this scope IMO